packages/viewer/src/modules/infobox/components/FeatureListCategoryItem.vue

```diff
@@ -98,6 +101,7 @@ function showContentAndScrollIntoView(event?: MouseEvent): void {
         <ZoomToExtentButton
             v-if="item.extent"
             :extent="item.extent"
+            :extent-projection="positionStore.projection"
```

Comment:
As discussed in DMs, let's make this extent projection props required in all cases (no default value provided, and error raised when absent)